tests/test_cli.py

```diff
@@ -152,6 +152,22 @@ def test_path_bad_permissions(path_kind, recipe_arguments):
                          options + recipe_arguments)
 
 
+@pytest.mark.parametrize('valid_dist',
+                         ('fc', 'fc26', 'el', 'el7', 'centos', 'centos7'))
+def test_dist_returns_value_on_valid_options(recipe_arguments, valid_dist):
+    options = ['--dist', valid_dist]
+    ctx = run.make_context('test_dist_returns_value_on_valid_options',
```

Comment:
See sclorg#87 – please wrap the context with `with` statement:

```python
with run.make_context(...) as ctx:
# or
ctx = run.make_context(*long_argument_list)
with ctx:
```

Same on the test below.